### PR TITLE
Fix #2941

### LINKED ISSUE #2941: pipeline("sentiment-analysis")() can't handle more than 2 sentences
# 🐛 Bug

## Information

Model I am using (Bert, XLNet ...): pipeline("sentiment-analysis")

Language I am using the model on (English, Chinese ...): English

The problem arises when using:
* [x] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x] my own task or dataset: (give details below)

## To reproduce

```
>>> from transformers import pipeline
>>> analyzer = pipeline('sentiment-analysis')
Downloading: 100%|██████████████████████████████| 230/230 [00:00<00:00, 146kB/s]
>>> analyzer(["OK"]*10)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../lib/python3.6/site-packages/transformers/pipelines.py", line 490, in __call__
    scores = np.exp(outputs) / np.exp(outputs).sum(-1)
ValueError: operands could not be broadcast together with shapes (10,2) (10,) 
>>> 
```

`

## Expected behavior

Getting 10 results

## Environment info

     
- `transformers` version: 2.5.0
- Platform: ubuntu 19.04
- Python version: 3.6
- PyTorch version (GPU?): 1.4.0 GPU
- Tensorflow version (GPU?): 1.14.0 GPU
- Using GPU in script?: Yes
- Using distributed or parallel set-up in script?: No

### PR DESCRIPTION
**Pull request by xxbidiao**
Reshaped score array to avoid `numpy` ValueError. This should allow the sentiment analyzer pipeline to run.

### CODE REVIEW

**Comment by xxbidiao** [comment]
Uh, I guess it should be `reshape(-1, 1)` instead of `reshape(-1,1)` regarding code quality issues, but I'm not sure whether it's the correct fix.

File: src/transformers/pipelines.py [hunk 1]
```diff
@@ -656,7 +656,7 @@ class TextClassificationPipeline(Pipeline):
 
     def __call__(self, *args, **kwargs):
         outputs = super().__call__(*args, **kwargs)
-        scores = np.exp(outputs) / np.exp(outputs).sum(-1)
+        scores = np.exp(outputs) / np.exp(outputs).sum(-1).reshape(-1,1)
```
Comment (julien-c) [inline src/transformers/pipelines.py:659]: You can also just do 
```suggestion
        scores = np.exp(outputs) / np.exp(outputs).sum(-1, keepdims=True)
```

File: src/transformers/pipelines.py [hunk 2]
```diff
@@ -656,7 +656,7 @@ class TextClassificationPipeline(Pipeline):
 
     def __call__(self, *args, **kwargs):
         outputs = super().__call__(*args, **kwargs)
-        scores = np.exp(outputs) / np.exp(outputs).sum(-1)
+        scores = np.exp(outputs) / np.exp(outputs).sum(-1, keepdims=True)
         return [{"label": self.model.config.id2label[item.argmax()], "score": item.max()} for item in scores]
```
Comment (julien-c) [inline src/transformers/pipelines.py:660]:
```suggestion
        return [{"label": self.model.config.id2label[item.argmax()], "score": item.max().item()} for item in scores]
```

File: src/transformers/pipelines.py (hunk 1 above)
Comment (xxbidiao) [reply src/transformers/pipelines.py]: Thanks for pointing this out!

**Pull request merged**